test/listen.test.ts

```diff
@@ -168,8 +164,6 @@ describe.skipIf(typeof EdgeRuntime === 'string' || typeof document !== 'undefine
 
       const {server, client} = await testSse(({channel}) => {
         channel!.send({event: 'channelError', data: {error: {description: 'Expected error'}}})
-        channel!.close()
```

Comment:
also not needed since an error thrown on the observable will call the observable cleanup, which will do the closing for us